android-lib/src/androidTest/java/com/kinvey/androidTest/store/datastore/CustomEndpointsTest.java

```diff
@@ -132,7 +132,7 @@ public void testCustomEndpointsNotExist() throws IOException, InterruptedExcepti
     */
     @Test
     public void testAsyncCustomEndpoints() throws IOException, InterruptedException {
-        AsyncCustomEndpoints endpoints = client.customEndpoints(GenericJson.class);
+        AsyncCustomEndpoints endpoints = (AsyncCustomEndpoints) client.customEndpoints(GenericJson.class);
```

Comment:
samea as the other PR, I don't think this cast is necessary

Reply:
Merged approved changes from `master`.